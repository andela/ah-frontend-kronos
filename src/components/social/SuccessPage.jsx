import React from 'react';

class SocialLoginSuccess extends React.Component {
  render() {
    return (
      <div>
        <div>
          <h1 className="app">
            {sessionStorage.getItem('user_email')}
            <br />
            Congrats!! You have successfully Logged in with your social Account.
          </h1>
        </div>
      </div>
    );
  }
}

export default SocialLoginSuccess;
