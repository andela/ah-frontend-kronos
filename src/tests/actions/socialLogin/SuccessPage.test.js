import React from 'react';
import { shallow } from 'enzyme';
import SuccessPage from '../../../components/social/SuccessPage';

// test to check if the App component matches the snapshot
describe('SuccessPage Component', () => {
  it('matches the snapshot', () => {
    const component = shallow(<SuccessPage />);
    expect(component).toMatchSnapshot();
  });
});
