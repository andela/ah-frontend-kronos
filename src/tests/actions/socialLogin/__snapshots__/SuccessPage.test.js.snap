// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SuccessPage Component matches the snapshot 1`] = `ShallowWrapper {}`;
